### PR TITLE
Switch from BuildTools packaging to Arcade packaging

### PR DESCRIPTION
**Pull request by ericstj**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -9,4 +9,7 @@
     <clear/>
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
   </packageSources>
+  <disabledPackageSources>
+    <clear />
```
Comment (joperezr) [inline NuGet.config]: NIT: Should we add a comment in here of why this is needed?

File: eng/Versions.props [hunk 2]
```diff
@@ -10,12 +10,14 @@
     <MicrosoftDotNetGenApiPackage>Microsoft.DotNet.GenAPI</MicrosoftDotNetGenApiPackage>
     <MicrosoftDotNetGenFacadesPackage>Microsoft.DotNet.GenFacades</MicrosoftDotNetGenFacadesPackage>
     <MicrosoftDotNetXUnitExtensionsPackage>Microsoft.DotNet.XUnitExtensions</MicrosoftDotNetXUnitExtensionsPackage>
+    <MicrosoftDotNetBuildTasksPackagingPackage>Microsoft.DotNet.Build.Tasks.Packaging</MicrosoftDotNetBuildTasksPackagingPackage>
```
Comment (weshaggard) [inline eng/Versions.props]: I know you are just following convention but we might want to reconsider the package name duplication now if the only place the name is used is in Tools.props.

**Comment by ericstj** [comment]
Fixed the version XML 🌮

**Comment by ericstj** [comment]
Test Linux arm64 Release Build

**Pull request merged**